packages/amplify-cli-core/src/cfnUtilities.ts

```diff
@@ -22,7 +23,11 @@ export function readCFNTemplate(filePath: string, options: Partial<typeof defaul
     if (options.throwIfNotExist === false) {
       return undefined;
     }
-    throw new Error(`No CloudFormation template found at ${filePath}`);
+    throw new AmplifyError('CloudFormationTemplateError', {
```

Comment:
is this user error or a fault? 
CLI manages these files not customer.

Reply:
This is what Praveen also pointed out, we do not have enough data to say with confidence, based on the number of distinct customers having this issue it seems to be a fault, but for the cases we've been on calls this file was manually renamed, going to mark this PR as draft until we gain more insights